Objective-C/CBLReplicatorConfiguration.h

```diff
@@ -136,30 +136,36 @@ __deprecated_msg(" Use [... initWithTarget:] and [config addCollection: config:]
 /**
  The heartbeat interval in second.
 
- The interval when the replicator sends the ping message to check whether the other peer is still alive. Set the value to zero(by default)
- means using the default heartbeat of 300 seconds.
+ The interval when the replicator sends the ping message to check whether the other peer is still alive.
+ Default heartbeat is 300 seconds.
```

Comment:
Link to the constant variable instead of mentioning the actual value here so when the value gets changed, there is only once place to update.

Example:

```
Default heartbeat is ``kCBLDefaultReplicatorHeartbeat``
```

Please check if the generated API doc links to the constant variable correctly.

Reply:
Same comments for the others.